x-pack/metricbeat/module/aws/dynamodb/manifest.yml

```diff
@@ -0,0 +1,42 @@
+default: true
+input:
+  module: aws
+  metricset: cloudwatch
+  defaults:
+    metrics:
+      - namespace: AWS/DynamoDB
+        statistic: ["Average"]
```

Comment:
I believe AWS allows users to add tags for DynamoDB, so we probably should add `tags.resource_type_filter` to collect tags by default.